bundles/org.openhab.binding.warmup/src/main/java/org/openhab/binding/warmup/internal/handler/WarmupThingHandler.java

```diff
@@ -77,6 +77,15 @@ protected void refreshFromServer() {
         }
     }
 
+    /**
+     *
+     * @param temperature value returned from the API as a String * 10. i.e. "215" = 21.5 degrees C
+     * @return the temperature as a {@link QuantityType}
+     */
+    protected State parseTemperature(@Nullable String temperature) {
+        return temperature != null ? parseTemperature(Integer.parseInt(temperature)) : UnDefType.UNDEF;
```

Comment:
`Float.parseInt(temperature)` can throw `NumberFormatException`. I would suggest to catch it and return UnDefType.UNDEF``